### PR TITLE
Add Copenhagen Click Pad

### PR DESCRIPTION
**Pull request by mini-ninja-64**
Add Copenhagen Click Pad

## Description
Added a keyboard and keymap for the Copenhagen Click Pad V1

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/copenhagen_click/click_pad_v1/rules.mk [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# MCU name
+#MCU = at90usb1286
```
Comment (fauxpark) [inline keyboards/copenhagen_click/click_pad_v1/rules.mk]:
```suggestion
```

File: keyboards/copenhagen_click/click_pad_v1/rules.mk [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+MCU = atmega32u4
+
+# Processor frequency.
```
Comment (fauxpark) [inline keyboards/copenhagen_click/click_pad_v1/rules.mk]: Lines 5 to 39 can be removed, as QMK provides these as defaults.

File: keyboards/copenhagen_click/click_pad_v1/click_pad_v1.c [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "click_pad_v1.h"
+
+// Optional override functions below.
```
Comment (drashna) [inline keyboards/copenhagen_click/click_pad_v1/click_pad_v1.c]: You can remove everything on this line and below.   They're not needed.

File: keyboards/copenhagen_click/click_pad_v1/config.h [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+// /*== customize breathing effect ==*/
+//   /*==== (DEFAULT) use fixed table instead of exp() and sin() ====*/
+//   #define RGBLIGHT_BREATHE_TABLE_SIZE 256      // 256(default) or 128 or 64
+//   /*==== use exp() and sin() ====*/
+//   #define RGBLIGHT_EFFECT_BREATHE_CENTER 1.85  // 1 to 2.7
+//   #define RGBLIGHT_EFFECT_BREATHE_MAX    255   // 0 to 255
+// #endif
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCING_DELAY 5
```
Comment (drashna) [inline keyboards/copenhagen_click/click_pad_v1/config.h]:
```suggestion
#define DEBOUNCE 5
```

File: keyboards/copenhagen_click/click_pad_v1/readme.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# CopenhagenClickPad-V1
+
+<!-- ![ClickPad](imgur.com image replace me!) -->
+
+A single switch macropad given out at the Copenhagen Click 2019 meetup.
+
+Keyboard Maintainer: [mini-ninja-64](https://github.com/mini-ninja-64)  
+Hardware Supported: Copenhagen Click Pad V1
```
Comment (fauxpark) [inline keyboards/copenhagen_click/click_pad_v1/readme.md]:
```suggestion
Hardware Supported: Copenhagen Click Pad V1  
```

File: keyboards/copenhagen_click/click_pad_v1/readme.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# CopenhagenClickPad-V1
+
+<!-- ![ClickPad](imgur.com image replace me!) -->
```
Comment (fauxpark) [inline keyboards/copenhagen_click/click_pad_v1/readme.md]: If you could please put an image here :)

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**